queries/capnp/highlights.scm

```diff
@@ -119,6 +129,8 @@
   "->"
 ] @punctuation.delimiter
 
+(data_hex) @symbol
```

Comment:
data doesn't sound like `@symbol`. `@symbol` is rather something like a named constant in the field of functional languages

Reply:
Capnp datas are unique, e.g.
```capnp
  dataField      @13 : Data    = 0x"62 61 72"; # "bar"
```

this is how you would write raw binary types, like b"" in python, so this *specific* leading 0x imo should be a special kind of highlight. what do you think?

Reply:
@number ?

Reply:
hmm, @number *just* for the 0x prefix is weird..but if thats convention sure, let me know

Reply:
@string.special works for me, let me update

Reply:
nevermind, i have that defined for the inner data string, it'd be weird to have them both be that